### PR TITLE
Add x-bte schema extension 

### PR DESCRIPTION
**Pull request by NikkiBytes**
Adding working schema and markdown file examples. 
Reference [ smartapi issue#236 ](https://github.com/SmartAPI/smartAPI/issues/236) for further insight.

### CODE REVIEW

**Review by newgene — COMMENTED** [review]
@NikkiBytes the schema file looks good, that examples.md file should be updated to reflect the `x-bte` extension examples?

**Comment by NikkiBytes** [comment]
@newgene thank you, I pushed the wrong edit, the `x-bte` examples are now there.

**Comment by NikkiBytes** [comment]
Ran a current validation with up to date data, and it ran as expected, no error in schema.

File: x-bte/smartapi_x-bte_examples.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        parameters:
+          fields: "exampleField"
+          size: 1000
+        predicate: "participates_in"
+        source: "infores:rhea"
+        response_mapping:
+          source_url: "http://example.com/response"
+```
+
+#### Example 2: `team` Field is Not an Array
```
Comment (newgene) [inline x-bte/smartapi_x-bte_examples.md:62]: @NikkiBytes this is not an x-bte example, right?

**Pull request merged**